atlasdb-client/src/main/java/com/palantir/atlasdb/table/description/render/StreamStoreRenderer.java

```diff
@@ -409,26 +410,32 @@ private void loadSingleBlockToOutputStream() {
                     line(StreamValueRow, " row = ", StreamValueRow, ".of(streamId, blockId);");
                     line("try {");
                     {
-                        line("os.write(getBlock(t, row));");
+                        line("byte[] block = getBlock(t, row);");
+                        line("if (block == null) {");
+                        line("throw new SafeRuntimeException(");
+                        line("        \"Block for stream not found. This is likely due to returning a stream "
+                                + " from a transaction and attempting to use it after it has been marked as"
```

Comment:
nit / annoying pedantry: The string building here will lead to 2 spaces between `stream` and `from`, we should just keep one of them

Reply:
Oops, this was not intentional